src/inverseRobot.js

```diff
@@ -8,6 +8,41 @@
 
 function inverseRobot(robot) {
   // write code here
+  const INVERSE_ROBOT = {};
+
+  // #1 use for in
+
+  for (const key in robot) {
+    if (INVERSE_ROBOT[robot[key]]) {
+      return null;
+    }
+    INVERSE_ROBOT[robot[key]] = key;
+  }
```

Comment:
The naming convention for variables in javascript is camelCase. Only global constants can have names with current style.
Checklist: 
2. [CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.